backend/src/Designer/Events/ProcessTaskIdChangedEvent.cs

```diff
@@ -0,0 +1,9 @@
+using MediatR;
+
+namespace Altinn.Studio.Designer.Events;
+
+public class ProcessTaskIdChangedEvent : INotification
+{
+    public string OldId { get; set; }
+    public string NewId { get; set; }
```

Comment:
Not sure what is best practice here, but would it be a good idea to extend the `TaskIdChange` model here to ensure that we get the correct property names and typings here?
Or potentially something like 
```csharp
public TaskIdChange TaskIdChange { get; set; }
```
instead of defining the properties directly here 🤔

Reply:
Be aware that my answer will be a bit opinionated 😄. 
But I think defining properties again follows the principles of DDD to keep the domain model and the DTO separate. TaskIdChange is a DTO that serves as a contract in the endpoint for communicating with the client. My argument is that this makes it easier and more maintainable to change the internal model, in this case the event, without affecting the contract or better to say the endpoint signature. 
Downside is that this design adds extra layer for mapping the DTO to events but in this pr it was pretty straightforward, I just used new keyword to "map" them.

Reply:
Thanks @mirkoSekulic! That makes sense to me 😊

Reply:
👏